src/utils/getExplorerLinks.ts

```diff
@@ -0,0 +1,77 @@
+import { SupportedChainId } from './chains'
+
+const ETHERSCAN_PREFIXES: { [chainId: number]: string } = {
```

Comment:
You can grab block explorer URLs from the `chain` object from wagmi, returned by `useActiveNetwork` #4 

```tsx
declare type Chain = {
    /** ID in number form */
    id: number;
    /** Human-readable name */
    name: string;
    /** Internal network name */
    network: string;
    /** Currency used by chain */
    nativeCurrency: NativeCurrency;
    /** Collection of RPC endpoints */
    rpcUrls: {
        [key: string]: RpcUrls;
        default: RpcUrls;
    };
    /** Collection of block explorers */
    blockExplorers?: {
        [key: string]: BlockExplorer;
        default: BlockExplorer;
    };
    /** Collection of contracts */
    contracts?: {
        ensRegistry?: Contract;
        multicall3?: Contract;
    };
    /** Flag for test networks */
    testnet?: boolean;
};
```

Reply:
Great - Used that instead